data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1615 @@
+			`	They leads you over to a panel in the main loading dock. "I know, it says 'Job Board' on it. I re-used the coding for our task list to improvise this mass-notification system." They smile in amusement. "When you are ready to go, just select the 'De-activate Array' option from the panel. That will notify everyone and begin the shutdown."`
+			`	They hand you a data crystal, and then a large case. "Here is an encrypted data crystal of all our data so far. The case has several sets, with the data stored in a variety of different mechanisms and encryptions. We do not know what happens when the station is activated or deactivated, so multiple formats for storing the data will give a better chance that at least some of it will remain intact to be delivered."`
+			`	They step back. "Well, I will return to the controls. Just give the signal when you are ready to leave. Embers light your path, Captain <first>."`
+
+
+
+mission "Remnant: Cognizance 36"
+	name "De-activate Ssil Vida Power"
+	description "Accept this to signal everyone to cut the power."
+	job
+	repeat
+	source "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 35: done"
+		has "Remnant: Ssil Vida Active"
+	on accept
+		event "remnant: ssil vida deactivation"
+		conversation
+			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta has once again returned to its prior, barren, form.`
+	on enter Postverta
+		fail
+
+
+
+mission "Remnant: Cognizance 37"
+	name "Activate Ssil Vida Power"
+	description "Accept this to signal everyone to restore the power."
+	job
+	repeat
+	source "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 35: done"
+		not "Remnant: Ssil Vida Active"
+	on accept
+		event "remnant: ssil vida activation"
+		conversation
+			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta is once again overflowing with life.`
+	on enter Postverta
+		fail
+
+
+
+mission "Remnant: Cognizance 38"
+	name "Take data to <planet>"
+	description "The rest of the Remnant need to know what is happening. Take this data to them on <destination>."
+	landing
+	source "Ssil Vida"
+	destination "Viminal"
+	to offer
+		has "Remnant: Cognizance 35: done"
+	on offer
+		event "remnant: cognizance salvage expansion"
```

Comment:
Is there somewhere in the conversation that the new availability of Chasers is noted?

Reply:
No, but I just added a little bit:

> He looks around, and then winks at you. "In the meantime, I have unlocked access to a few more bits of rarer Korath salvage, including access to captured Chasers.

Slightly expanded the second last sentence.